src/QMCWaveFunctions/Fermion/DelayedUpdateBatched.h

```diff
@@ -645,11 +645,11 @@ class DelayedUpdateBatched
     //std::copy_n(Ainv[rowchanged], norb, V[delay_count]);
     compute::BLAS::copy_batched(blas_handle, norb, invRow_mw_ptr, 1, V_row_mw_ptr, 1, nw);
     // handle accepted walkers
-    // the new Binv is [[X Y] [Z sigma]]
+    // the new Binv is [[X y] [z sigma]]
```

Comment:
So is the meaning here that y and z are vectors and X a matrix?

Reply:
Upper case is a matrix and lower case is a vector. Names are aligned with the paper I'm working on.